### PR TITLE
extensions/khr/ray_tracing_pipeline: Pass indirect SBT regions as single item

### PR DESCRIPTION
**Pull request by MarijnS95**
In an identical fashion to commit 84624fd ("ray_tracing_pipeline: Pass SBT regions as reference instead of slice (#350)") `cmd_trace_rays_indirect()` also only needs a pointer to a single `StridedDeviceAddressRegionKHR` structure.  After all no length is ever passed to the API anywhere, and this could also lead to users passing empty slices, or passing too many elements that are never used.

Clear up the confusion by replacing the slice argument with a direct borrow of the struct.

### CODE REVIEW

File: ash/src/extensions/khr/ray_tracing_pipeline.rs [hunk 1]
```diff
@@ -51,20 +51,24 @@ impl RayTracingPipeline {
         &self,
         deferred_operation: vk::DeferredOperationKHR,
         pipeline_cache: vk::PipelineCache,
-        create_info: &[vk::RayTracingPipelineCreateInfoKHR<'_>],
+        create_infos: &[vk::RayTracingPipelineCreateInfoKHR<'_>],
         allocation_callbacks: Option<&vk::AllocationCallbacks<'_>>,
-    ) -> VkResult<Vec<vk::Pipeline>> {
-        let mut pipelines = vec![mem::zeroed(); create_info.len()];
-        (self.fp.create_ray_tracing_pipelines_khr)(
+    ) -> Result<Vec<vk::Pipeline>, (Vec<vk::Pipeline>, vk::Result)> {
```
Comment (Ralith) [inline ash/src/extensions/khr/ray_tracing_pipeline.rs:56]: This seems to be unrelated to the change described by the commit.

Comment (MarijnS95) [reply ash/src/extensions/khr/ray_tracing_pipeline.rs]: Looks like I did end up making a mess after all when rebasing these changes off of that PR, and failing to properly reload the file without those changes in my editor. Fixed now!

**Review by Ralith — APPROVED** [review]
Nice catch!
